values.yaml

```diff
@@ -74,9 +74,11 @@ serviceAccount:
 
 rbac:
   create: true
+  installJobRoleName: "timothy-install-job"
 
 image:
-  name: "tonicai/timothy"
+  webName: "tonicai/timothy"
+  installName: "tonicai/timothy_install"
```

Comment:
Instead of renaming `name` can we stick these properties on the `timBot` key (and maybe a better name for it as well lol)